packages/components/modules/notifications/native/NotificationBellWithBadge/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { Theme } from '@baseapp-frontend/design-system/styles/native'
+
+import { StyleSheet } from 'react-native'
+
+export const createStyles = (theme: Theme) =>
+  StyleSheet.create({
+    badge: {
+      position: 'absolute',
+      top: -10,
+      right: -10,
+      backgroundColor: theme.colors.error.main,
+      color: theme.colors.error.contrast,
+      ...theme.typography.caption,
+      fontWeight: 700,
```

Comment:
this also doesn't look right, overriding the DS typography, can we double check? maybe the designs are wrong